docs/book/v3/application-integration/usage-in-a-laminas-mvc-application.md

```diff
@@ -0,0 +1,181 @@
+
+<pre class="language-php" data-line="3,8"><code>
+namespace Application;
+
+use Laminas\ServiceManager\AbstractFactory\ConfigAbstractFactory;
+
+return [
+    'controllers' => [
+        'factories' => [
+            Controller\IndexController::class => ConfigAbstractFactory::class,
+        ],
+    ],
+    // …
+];
+</code></pre>
+
+The example uses the [config factory from laminas-servicemanager](https://docs.laminas.dev/laminas-servicemanager/config-abstract-factory/) which allows any string to be used to fetch a service from the application service container, like the name of the configured cache: `default-cache`.
+
+This means that the factory [searches for an appropriate configuration](https://docs.laminas.dev/laminas-servicemanager/config-abstract-factory/#configuration) to create the controller and to resolve the constructor dependencies for the controller class.
+
+### Add Factory Configuration For Controller
+
+Extend the module configuration file to add the configuration for the controller.
+Use the name of the cache (`default-cache`), which was previously defined in the configuration of the caches, to retrieve the related cache storage instance:
+
+<pre class="language-php" data-line="11-15"><code>
+namespace Application;
+
+use Laminas\ServiceManager\AbstractFactory\ConfigAbstractFactory;
+
+return [
+    'controllers' => [
+        'factories' => [
+            Controller\IndexController::class => ConfigAbstractFactory::class,
+        ],
+    ],
+    ConfigAbstractFactory::class => [
+        Controller\IndexController::class => [
+            'default-cache',
+        ],
+    ],
+    // …
+];
+</code></pre>
+
+## Using Multiple Caches
+
+The use more than one cache backend, the factory `Laminas\Cache\Service\StorageCacheAbstractServiceFactory` allows to define multiple cache storages.
+
+Extend the cache configuration in `config/autoload/global.php` and add more cache adapters:
+
+<pre class="language-php" data-line="9-14"><code>
+return [
+    'caches' => [
+        'default-cache' => [
+            'adapter' => Laminas\Cache\Storage\Adapter\Filesystem::class,
+            'options' => [
+                'cache_dir' => __DIR__ . '/../../data/cache',
+            ],
+        ],
+        'secondary-cache' => [
+            'adapter' => Laminas\Cache\Storage\Adapter\Memory::class,
+        ],
+        'blackhole' => [
+            'adapter' => Laminas\Cache\Storage\Adapter\BlackHole::class,
+        ],
+        // …
+    ],
+    // ...
+];
+</code></pre>
+
+MISSING: **Installation Requirements**
+Make sure that the used storage adapters are installed:
+```bash
+$ composer require laminas/laminas-cache-storage-adapter-memory laminas/laminas-cache-storage-adapter-blackhole
+```
+
+### Change Used Adapter for Controller
+
+To use a different cache adapter for the controller, change the related module configuration and use one of the previously defined names:
```

Comment:
@boesing 
With simple use, I mean exactly this step.

Writing another factory by hand only distracts from the topic in this description and is annoying and cumbersome.